pkg/kubernetes/kubectl.go

```diff
@@ -128,14 +128,15 @@ func (k Kubectl) Get(namespace, kind, name string) (map[string]interface{}, erro
 }
 
 // Apply applies the given yaml to the cluster
-func (k Kubectl) Apply(yaml string) error {
+func (k Kubectl) Apply(yaml, namespace string) error {
 	if err := k.setupContext(); err != nil {
 		return err
 	}
 
 	reader := bufio.NewReader(os.Stdin)
-	fmt.Printf(`!!! Applying to cluster '%s' at '%s' using context '%s'.
-!!! Please type 'yes' to perform: `,
+	fmt.Printf(`Applying to namespace '%s' of cluster '%s' at '%s' using context '%s'.
+Please type 'yes' to perform: `,
+		alert(namespace),
```

Comment:
When you run `kubectl apply` do you not specify `--namespace`?

Reply:
I am not sure if this is even possible. Anyways, I hard set the namespace on every object to the one of spec.json (yes, this does not allow other namespaces than the one in spec, will address later)

Reply:
Hmm, thought you could, but must have imagine it.